src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -23,6 +23,8 @@
 
 #include <QMimeData>
 #include <QtMath>
+#include <QWheelEvent>
+#include <QTimer>
```

Comment:
...but rather here.

Also, I think it would be better to drop `<QWheelEvent>` from here, and instead `#include <QtEvents>` in `notationviewinputcontroller.h` (and then remove the `class QMouseEvent;` forward declaration from that file). This is because many kinds of Q…Events are used in the declarations and definitions of members of NotationViewInputController. 

Final request: I prefer to keep the includes alphabetically sorted within the "groups".

Reply:
Same for QQuickItem?

Reply:
Nah, this looks fine to me now. 

Regarding the sorting, I meant sorting it like this:
```c++
#include <QMimeData>
#include <QQuickItem>
#include <QTimer>
#include <QtMath>
```
(By the way, Qt Creator has an option to sort lines in Edit > Advanced, which I find very convenient)

Reply:
Of course, didn't get my ABC right...